docs/endpoints/post-identity-map-v3.md

```diff
@@ -109,9 +109,11 @@ For details, and code examples in different programming languages, see [Encrypti
 The response is encrypted only if the HTTP status code is 200. Otherwise, the response is not encrypted.
 :::
 
-A successful decrypted response returns the UID2s for the specified email addresses, phone numbers, or their respective hashes. The sequence in the response matches the sequence in the request.
+A successful decrypted response returns the current UID2s, previous UID2s (where applicable) and refresh timestamps for the specified email addresses, phone numbers, or their respective hashes. 
 
-Identifiers that cannot be mapped to a UID2 are mapped to an error object with the reason for unsuccessful mapping. An unsuccessful mapping occurs if the identifier is considered invalid or if the identifier has opted out from the UID2 ecosystem. In these cases, the response status is still "success".
+The response arrays preserve the order of input arrays. Each element in the response array maps directly to the element at the same index in the corresponding request array. This ensures that results can be reliably associated with their corresponding inputs based on array position.
```

Comment:
It'd be nice to get an example for this. We have an example input above and an example output below, but they do not actually match and do not illustrate the point.

Reply:
Thanks, added an example